addons/test_website/static/tests/tours/image_link.js

```diff
@@ -54,7 +52,7 @@ wTourUtils.registerWebsitePreviewTour('test_image_link', {
     }, {
         content: "enter email URL",
         trigger: "#oe_snippets we-customizeblock-options:has(we-title:contains('Image')) we-input:contains(Your URL) input",
-        run: "edit mailto:test@test.com",
+        run: "edit mailto:test@test.com && click body",
```

Comment:
As discussed for multiple days with PIPU and ARD, this is wrong and should not be needed.

Can you either revert all those `&& click body` you added here, or add `// TODO: Remove me`

This is symptomatic of a problem in Hoot mechanism IMO, the next step is clicking somewhere, it should trigger the blur and so on, the click on body should not exists and should not be needed.

@detrouxdev Had a first look, and it's probably because hoot is not handling iframes correctly.

We did not see the code owner ping on time to comment on this PR.

Related:
https://github.com/odoo/odoo/pull/162031#discussion_r1575844952
https://github.com/odoo/odoo/pull/162031#discussion_r1572330368
And other review points

Maybe @detrouxdev can help here?